docs/book/src/reference/compiler_intrinsics.md

```diff
@@ -78,14 +78,14 @@ ___
 __state_load_quad(key: b256, ptr: raw_ptr)
 ```
 
-**Description:** Reads a `b256` from storage at key `key` and stores it in memory at address `raw_ptr`
+**Description:** Reads `slots` number of slots (`b256` each) from storage starting at key `key` and stores them in memory starting at address `ptr`
```

Comment:
Looks like `__state_load_quad`'s parameters are not updated: ![image](https://user-images.githubusercontent.com/2316541/209175617-e9bd8e95-5f70-4728-a56f-b24668d73c48.png)

Reply:
Addressed!